projects/managed-hubs/roles.md

```diff
@@ -56,32 +52,11 @@ People in these roles are generally affiliated with 2i2c.
 - Debug and resolve major issues with a hub that require intervention from a Hub Engineer
 - Perform open source development on technologies that are in use by the hubs
 
-## Support Steward
```

Comment:
This section is mostly duplicated in our dedicated support page now, so removing it